introduction_to_amazon_algorithms/imageclassification_mscoco_multi_label/Image-classification-multilabel-lst.ipynb

```diff
@@ -42,9 +42,7 @@
   },
```

Comment:
<p>This assumes a conda kernel is being used, I guess? Does using pip no longer work? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
the `pycocotools` package that pip would install with the old command, would result in a library with dependencies that were failing some commands further down in the notebook. I didn't manage to fix that with `pip` and after reading about it more online, one recommendation was to use that command to install the library. 

Also, the kernel specs of this notebook are (see the [raw file](https://raw.githubusercontent.com/aws/amazon-sagemaker-examples/main/introduction_to_amazon_algorithms/imageclassification_mscoco_multi_label/Image-classification-multilabel-lst.ipynb)):

```
"kernelspec": {
   "display_name": "conda_python3",
   "language": "python",
   "name": "conda_python3"
  }
```

so I thought this change was acceptable.